ui/src/app/views/workspace/component.ts

```diff
@@ -90,6 +91,7 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
   cohortDescriptionComparator = new CohortDescriptionComparator();
```

Comment:
Not removing some of these as I'm not sure what may/may not still be in use for Bru's interdependent work